modules/manage/partials/mountable-topics.adoc

```diff
@@ -0,0 +1,203 @@
+```
+
+You can also rename the topic as you mount it to the target cluster:
+
+```
+rpk topic mount <namespace>/<source-topic-name> --to <namespace>/<new-topic-name>
+```
+--
+Admin API::
++
+--
+To mount a topic to a target cluster using the Admin API, you must first create an inbound topic migration. The API returns the migration ID upon creation. Then, you make an API request with the migration ID to carry out the migration in steps: prepare, execute, and cut over. You must wait for each step to complete and for the <<monitor-progress,migration state>> to update before proceeding to the next step.
+
+. Make a PUT request to the `/migrations` endpoint to create a topic migration. Specify the names of the topics in the request body:
++
+```
+curl -X PUT http://localhost:9644/v1/migrations -d {
+  "migration_type":"inbound", 
+  "topics": [
+    {
+      "source_topic": {"ns": "kafka", "topic": "<source-topic-1-name>"}, 
+      "alias": {"ns": "kafka", "topic": "<new-topic-1-name>"}
+    }, 
+    {
+      "source_topic": {"ns": "kafka", "topic": "<source-topic-2-name>"}
+    }, 
+    {
+      "source_topic": {"ns": "kafka", "topic": "source-topic-3-name"}, 
+      "alias": {"ns": "kafka", "topic": "<new-topic-3-name>"}
+    }
+  ], 
+  "consumer_groups": []
+}
+```
++
+* `ns` is the topic namespace. This field is optional. Default value: `kafka`
+* To rename the topic in the target cluster, use the optional `alias` object in the request body. In the example, topics 1 and 3 are given new names in the target cluster, while topic 2 retains its original name.
+* `consumer_groups` is required.
++
+If successful, the response returns the ID of the newly-created migration. Use the ID in subsequent API calls to run the migration.
+
+. When preparing a topic for migration, Redpanda recreates the topic in a disabled state in the target cluster. The cluster allocates partitions but does not add log segments yet. Topic metadata is populated from the topic manifest found in object storage.
++
+Make a `POST /v1/migrations/\{id}/?action=prepare` request to prepare the migration:
++
+```
+curl -X POST http://localhost:9644/v1/<migration-id>/?action=prepare
+```
+
+. When you execute the migration, the target cluster starts the partitions with log segments downloaded from object storage.
++
+Make a `POST /v1/migrations/\{id}/?action=execute` request to execute the migration:
++
+```
+curl -X POST http://localhost:9644/v1/<migration-id>/?action=execute
+```
+
+. Finally, the migration can be deleted. The target cluster starts to handle produce and consume workloads.
++
+To cut over after executing the migration:
++
+```
+curl -X POST http://localhost:9644/v1/<migration-id>/?action=finish
+```
+
+--
+
+======
+
+== Cancel a migration
+
+You can cancel a topic migration by running this command:
+
+```
+
+```
+
+You can only cancel a migration that is not in the following final <<monitor-progress,states>>:
+
+- `cut_over`
+- `finished`
+
+== Monitor progress
```

Comment:
When or why might users encounter errors? Can and should they retry?

Reply:
I can think of the following situations:
- attempting to mount a topic that does not exist in the cloud storage
- attempting to mount a topic that is already mounted to this or another cluster
- any failures, such as tiered storage availability problems or multiple redpanda nodes going down.

All operations are retried indefinitely, so it's really unlikely that cancelling and restarting a migration would help. If there is any underlying problem fixing it should help without restarting.